specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/examples/GetCatalog.json

```diff
@@ -59,20 +59,16 @@
             "P1Y",
             "P3Y"
           ],
-          "billingPlans": [
-            {
-              "P1Y": [
-                "Upfront",
-                "Monthly"
-              ]
-            },
-            {
-              "P3Y": [
-                "Upfront",
-                "Monthly"
-              ]
-            }
-          ],
+          "billingPlans": {
```

Comment:
These were existing examples which had a syntax issue. Fixing this so that model validation would pass.